egs/tedlium/s5/local/score_sclite.sh

```diff
@@ -68,8 +78,8 @@ fi
 if [ $stage -le 1 ]; then
   # Remove some stuff we don't want to score, from the ctm.
   for x in $dir/score_*/ctm; do
-    cat $x | grep -v -E '"\[BREATH|NOISE|COUGH|SMACK|UM|UH\]"' | \
-      grep -v -E '"!SIL|\<UNK\>"' > ${x}.filt || exit 1;
+    cat $x | grep -v -w -E '\[BREATH|NOISE|COUGH|SMACK|UM|UH\]' | \
+      grep -v -w -E '!SIL|<unk>' > ${x}.filt || exit 1;
```

Comment:
I will do this; it is more robust in case someone copies this code without thinking enough.